releasenotes/README.md

```diff
@@ -12,9 +12,41 @@ If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx
+
+  Check out the [status page](https://github.com/salesforcecli/status) to get a bird’s eye view of the project. 
+
+---
+ 
+* NEW: Get more granular information about the code coverage results of your Apex test runs with the new `--detailedcoverage` parameter of the `force:apex:test:run` command. We've also greatly improved the code coverage results and output of the `force:apex:test:run` and `force:apex:test:report` commands. Check out the [Clearer Apex Commands](https://developer.salesforce.com/blogs/2021/04/clearer-apex-commands.html) blog post for more information. 
+
+* CHANGE: We changed the URLs we use to distribute the TAR files for installing Salesforce CLI. 
+
+    This table lists the unversioned URLs for the TAR files (`.tar.gz` or `.tar.xz`) for each operating system. When we release a new version of Salesforce CLI every week, we also update these URLs so they point to the most up-to-date version. 
+    
+    | Operating System | TAR Files                |
+    |------------------|--------------------------|
+    |Linux| [sfdx-linux-x64.tar.gz](https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-linux-x64.tar.gz)|
+    ||[sfdx-linux-x64.tar.xz](https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-linux-x64.tar.xz)|
+    ||[sfdx-linux-arm.tar.gz](https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-linux-arm.tar.gz)|
+    ||[sfdx-linux-arm.tar.xz](https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-linux-arm.tar.xz)|
+    |macOS|[sfdx-darwin-x64.tar.gz](https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-darwin-x64.tar.gz)|
+    ||[sfdx-darwin-x64.tar.xz](https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-darwin-x64.tar.xz)|
+    |Windows*|[sfdx-win32-x64.tar.gz](https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-win32-x64.tar.gz)|
+    | |[sfdx-win32-x64.tar.xz](https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-win32-x64.tar.xz)|
+    | |[sfdx-win32-x86.tar.gz](https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-win32-x86.tar.gz)|
+    ||[sfdx-win32-x86.tar.xz](https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-win32-x86.tar.xz)|
+
+    * WINDOWS NOTE: Installing Salesforce CLI with a TAR file on Windows requires a separate program, such as 7Zip, to extract the contents. We highly recommend you use the [Windows installers](https://developer.salesforce.com/tools/sfdxcli).
+
+    We also removed the `install` script from the TAR files. You had to have administrator privileges to run the script, and it wasn’t necessary to install Salesforce CLI anyway. We now recommend that you simply unpack the TAR file into a directory of your choice and update your PATH environment variable appropriately. 
```

Comment:
These two things sound like breaking changes (changing the URLs for the tarballs and removing the install script)

This can't just be merged in and quietly mentioned in release notes -- remember what happened back on #801 ?

Reply:
If customers are using these archives in CI/CD and expect install script to be there, it will break.

Reply:
Thanks for the note, David.  It prompted me to update the rel notes to clarify that the existing URLs are still there, will continue to be there for a while, still contain the install script, and so existing scripts shouldn't break.  But it's also true that, to install the most up to date releases, scripts will need to use the new URLs and not use the old install script.  We'll be posting a pinned issue about this soon. Stay tuned!

Reply:
Thanks -- I saw the update to the language -- perfect